### DISCUSSION TITLE
Welcome to LWJGL discussions!

### DISCUSSION THREAD

**Discussion by apostolos** [LWJGL/lwjgl3#606] [General]
## 👋 Welcome!

We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and be open-minded. Remember that this is a community we
  build together 💪.

**Comment by kostaskougios** [comment]
Hi all, new to LWJGL. I am trying to compile the samples from github.com:LWJGL/lwjgl3-demos.git but mvn can't download the SNAPSHOT dependencies.

```
        <lwjgl.version>3.3.0-SNAPSHOT</lwjgl.version>
        <joml.version>1.10.1-SNAPSHOT</joml.version>


```

If I set those versions to 3.2.3 & 1.10.1 it downloads the dependencies ok but I get a lot of compilation errors.
I've checked the LWJGL source code as well, there doesn't seem to be anything above v3.2.3 at the moment.

How can I make the sample project work?

**Reply by SWinxy** [reply]
You need to be using the proper maven coordinates. If you clones the repo it should be in the pom. If you just copied some stuff over here's the stuff you need:
```xml
<repositories>
	<repository>
		<id>sonatype-snapshots</id>
		<url>https://oss.sonatype.org/content/repositories/snapshots</url>
		<releases><enabled>false</enabled></releases>
		<snapshots><enabled>true</enabled></snapshots>
	</repository>
</repositories>
```

**Reply by kostaskougios** [reply]
Hi, thanks, I just cloned the demo git repo and run ./mvnw package but it doesn't work.

I changed pom.xml according to your example above, and managed to compile. I ended with these repos:

```
    <repositories>
        <repository>
            <id>sonatype-snapshots</id>
            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
            <releases><enabled>false</enabled></releases>
            <snapshots><enabled>true</enabled></snapshots>
        </repository>
        <repository>
            <id>swt-repo</id>
            <url>http://maven-eclipse.github.io/maven</url>
        </repository>
    </repositories>
```

and this
```
        <joml.version>1.10.1</joml.version>

```

and it compiles now.

**Comment by cpope9141** [comment]
Does anyone know when LWJGL 3 will include the latest Vulkan SDK (1.2+)?

**Reply by SWinxy** [reply]
When 3.3.0 is released. It's already in the early access snapshot builds, if you want to try that out (it's quite stable already).

**Comment by cpope9141** [comment]
I am trying the 3.3.0 early access build and am seeing an issue with Assimp importing joint transformations. Do I report issues for early access builds here on github?
